### DISCUSSION TITLE
Literally everything I've been looking for!

### DISCUSSION THREAD

**Discussion by khedrick2003** [shaked6540/YoutubePlaylistDownloader#184] [General]
CPU usage is low while using it (unlike other video/audio downloaders for Windows). Now I can enjoy downloading my music without the hassle of the program itself crashing or acting up. THANK YOU SO MUCH!

**Comment by ghost** [comment]
Agreed. Thank you shaked6540 for making the best, most user-friendly and free YT Downloader I've ever used. You are a hero to society!
